transport/src/main/java/io/netty/channel/nio/AbstractNioChannel.java

```diff
@@ -198,20 +197,6 @@ private void clearReadPending0() {
     }
 
     /**
-     * Return {@code true} if the input of this {@link Channel} is shutdown
-     */
-    protected boolean isInputShutdown() {
```

Comment:
I think it's fine because it was broken before and it affects the transport implementations only.